tools/run_docker.sh

```diff
@@ -17,7 +19,7 @@ do
 done
 
 echo Creating dist folder with privileges of host-machine user
-mkdir dist  # This is required to be created with host-machine user privileges
+mkdir -p dist  # This is required to be created with host-machine user privileges
```

Comment:
This is not needed. `--path` argument only takes effect if you create subpaths, like `dist/subfolder` and `dist` doesn't exist. This won't change anything for a single atom.

Reply:
I've added it for running `./tools/run_docker.sh "aiohttp"` locally.
On travis the change makes no harm but I've been tired to run `rm -rf dist` before `run_docker.sh` every time.

Reply:
Yea, I see. But I'm talking about the `-p` argument :)

Reply:
I'm also talking about `-p`.
With it `mkdir` doesn't create `dist` folder if it already exists.